homeassistant/strings.json

```diff
@@ -95,6 +95,7 @@
         "usb_path": "USB device path",
         "access_token": "Access token",
         "api_key": "API key",
+        "app_specific_password": "App-specific password",
```

Comment:
Thanks, but this PR cannot stand on its own.

This is basically adding an unused string in this PR.

Reply:
Good to know, thanks!